Blog post on bulk actions (#294)
DFE-Digital/bat-design-history: app/posts/register-trainee-teachers/2020-12-29-bulk-actions-v1.md

---
title: Bulk actions on a group of records
description:
date: 2020-12-18
---
{% from "screenshots/macro.njk" import appScreenshots with context %}

Providers have told us they often have a group of trainees to recommend for QTS at the same time - all with the same programme and training details.

In DTTP and with our current service, this means opening each record individually to recommend it for QTS - providing the same date each time. If a provider had 50 trainees, this could take half a day or more.

We’ve also heard that providers will add a bunch of records as draft, check them as a group, and then submit them all for TRNs in one go. We’re not sure this same behaviour will happen once we launch the service as we hope there’ll be less need to check records separately.

We can help support these two processes by letting providers perform actions in bulk on a group of records.

Initial assumptions:

> * We’ll need all details to be the same for the selected group - so for QTS this means the dates will need to be the same

Reviewer: The reader might not be sure what date we’re specifically referring to here?

```suggestion
* All details will need to be the same for the selected group, for example the entire group will be recommended for QTS on the same date
```
